### ISSUE TITLE
Modifiers for `perceive-object` still symbols, not keywords

### ISSUE THREAD

**Issue by fairlight1337** [cram2/cram_plans#2]
Since the designator properties became keywords rather than symbols as well as e.g. the identifiers for objects in bullet (for example `:static-plane` instead of `'static-plane`), the modifiers for plans like `perceive-object` should also become keywords.

It comes up [here](https://github.com/cram2/cram_plans/blob/master/cram_plan_library/src/perceive-object.lisp#L37), [here](https://github.com/cram2/cram_plans/blob/master/cram_plan_library/src/perceive-object.lisp#L73), [here](https://github.com/cram2/cram_plans/blob/master/cram_plan_library/src/perceive-object.lisp#L88), and [here](https://github.com/cram2/cram_plans/blob/master/cram_plan_library/src/perceive-object.lisp#L101). The same accounts for the respective [goal declarations](https://github.com/cram2/cram_plans/blob/master/cram_plan_library/src/goal-declarations.lisp).

Meaning

``` lisp
(def-goal (perceive-object all ?obj-desig) ...)
```

becomes

``` lisp
(def-goal (perceive-object :all ?obj-desig) ...)
```

Any objections, @gaya- and @mpomarlan?

**Comment by gaya-** [comment]
Yes, let's make those keywords.

**Comment by fairlight1337** [comment]
Okay, I'll refactor them later today.

**Comment by gaya-** [comment]
Solved in #3
